src/main/java/org/takes/rq/RqOnce.java

```diff
@@ -33,9 +37,9 @@
  * <p>The class is immutable and thread-safe.
  *
  * @since 0.36
- * @todo #918:30min Please use {@link org.cactoos.scalar.Sticky} decorator
- *  class so that multiple calls to body() would produce the same cached result
- *  rather than throwing an exception.
+ * @todo #999:30min Please use {@link org.cactoos.text.Sticky} decorator
+ *  class instead of inner anonymous class below, which prevents input
```

Comment:
@andreoss This seems to be fairly simple to do, couldn't you do this in this same PR?

Reply:
@paulodamaso No, this class is not yet released.